### ISSUE TITLE
Accessibility on Mac platform - dialog box is not accessible through keyboard

### ISSUE THREAD

**Issue by bharathpalaksha** [microsoft/azuredatastudio#23684]
- Azure Data Studio Version: 1.44.1
- Extension - Azure Cosmos DB Migration for MongoDB
- OS Version: Mac

This issue is raised on ADS because it is present on save dialog accessibility
vscode.window.showSaveDialog - This dialog box has the issue
Steps to Reproduce:

1.  Watch the below video for reproducing steps

https://github.com/microsoft/azuredatastudio/assets/342603/021c721e-cb9a-4fa6-9aee-23886f0bd4b5

2.

Does this issue occur when all extensions are disabled?: Yes/No
